src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,21 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
```

Comment:
The class name 'HelloWorld' is not descriptive of its purpose. Consider renaming the class to something more indicative of its role, such as 'FigureDemo' or 'FigureApplication'.